packages/compass-explain-plan/src/components/explain-tree/explain-tree-data.ts

```diff
@@ -75,9 +75,12 @@ const parseExplainTree = (
 const extractHighlights = (stage: Omit<ExplainTreeNodeData, 'highlights'>) => {
   switch (stage.name) {
     case 'IXSCAN':
+    case 'EXPRESS_IXSCAN':
```

Comment:
Based on the ticket, it sounds a bit like we should add both `EXPRESS_CLUSTERED_IXSCAN` and `CLUSTERED_IXSCAN` here as well?

Reply:
Yes, working on that right now :)